test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities
+./test_scripts/API/RAI/002_ATF_N_MissedAllParamaters_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_missing_all_parameters
+./test_scripts/API/RAI/003_ATF_N_Invalid_Json_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_invalid_json_message
+./test_scripts/API/RAI/004_ATF_N_correlationID_IsMissed_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_corrID_missing
+./test_scripts/API/RAI/005_ATF_N_correlationID_IsNonexistent_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_corrID_isn't_existing
+./test_scripts/API/RAI/006_ATF_N_correlationID_IsWrongType_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_corrID_is_wrong_type
+./test_scripts/API/RAI/007_ATF_N_correlationID_IsNegative_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities__json_message_corrID_is_negative
+./test_scripts/API/RAI/008_ATF_N_method_IsMissed_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_missing
+./test_scripts/API/RAI/009_ATF_N_method_IsNotValid_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_not_valid
+./test_scripts/API/RAI/010_ATF_N_method_IsOtherResponse_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_other_response
+./test_scripts/API/RAI/011_ATF_N_method_IsEmpty_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_empty
+./test_scripts/API/RAI/012_ATF_N_method_IsWrongType_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_wrong_type
+./test_scripts/API/RAI/013_ATF_N_method_IsInvalidCharacter_Space_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_invalid_character_space
+./test_scripts/API/RAI/014_ATF_N_method_IsInvalidCharacter_Tab_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_invalid_character_tab

```

Comment:
@OVikhrov, this is the idea of test. When SDL receives such message from HMI it rejects it and this is test case when HMI doesn't respond to UI.GetCapabilities.
According to requirement, case: "SDL does NOT receive value of "steeringWeelLocation" parameter via UI.GetCapabilities_response from HMI"

According to "requrements for validation "method" on whitespaces." there is no specific requirement. It is based on protocol layer verification.
According to sdl_core/src/components/interfaces/Json_HMI_message_specification.txt method is
"A String containing the name of the method to be invoked." and "This member MUST NOT exist if there was an error invoking the method."
Also in https://smartdevicelink.com/en/docs/hmi/master/websocket-transport/:
"A String containing the information of the method to be invoked. The format is [componentName].[methodName]."
In this way different verification for json format with wrong method are applied. You will see also scripts that check wrong method with:
method is missing; method is not valid; method is from other request; method is empty; method has wrong type; method has invalid characters: only space, /n, /t